pkg/fab/discovery/discovery_test.go

```diff
@@ -9,6 +9,7 @@ package discovery
 import (
 	"context"
 	"fmt"
+	discclient "github.com/hyperledger/fabric-sdk-go/internal/github.com/hyperledger/fabric/discovery/client"
```

Comment:
imports aren't sorted - this should be moved into the next group.

Reply:
I've just run goimports on that file, fixed.